airflow/executors/kubernetes_executor.py

```diff
@@ -79,7 +81,7 @@ class KubernetesJobWatcher(multiprocessing.Process, LoggingMixin):
 
     def __init__(
         self,
-        namespace: str | None,
+        namespace: str,
         multi_namespace_mode: bool,
```

Comment:
i think this can be removed now, along with its attr

Reply:
Nice catch! Address with https://github.com/apache/airflow/pull/28047/commits/742468ef37a0725d657ac4790f9c3daf22df9766